scripts/release.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python
```

Comment:
This file is "copied" into this repository just like it'll be copied with the others. Copying it means there's one less thing to install to maintain these projects. Because it only uses things in Python 3.6+ stdlib, it doesn't need a virtual env or anything like that and doesn't have to be run from inside a Docker container.